docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -7263,8 +7263,11 @@ function mass_theme_preprocess_breadcrumb(&$variables) {
   $variables['borderless'] = FALSE;
 
   if ((bool) $collection_page) {
-    $variables['borderless'] = (bool) ($collection_page->field_short_description->value ||
-      $collection_page->field_header_color->value);
+    $variables['borderless'] = (bool) (
+      $collection_page->field_short_description->value ||
+      $collection_page->field_header_color->value
+      )
+    );
```

Comment:
@beto-aveiga It looks like you're missing a parenthesis

Reply:
Many thanks @tom-fleming !